src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +313,39 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
         except Py4JError:
             traceback.print_exc()
+            
+    def start(self, sqlCtx):
+		try:
+			prevGlobalMLContext = MLContext.globalMLContext
+		except AttributeError:
+			pass
+		self.numTabs = 0
+		self.sqlCtx = sqlCtx
```

Comment:
We can just use `self.sqlCtx = SQLContext.getOrCreate(sc)`, assuming a `from pyspark.sql import SQLContext` import statement up top.  That way, we can get rid of the need to explicitly create and pass around a SQLContext object (which is going away anyways in 2.0).